client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -50,16 +50,16 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = ( {
 						</StepperWizard>
 						{ siteVertical && (
 							<div className="onboarding-block__footer">
-								<Button className="onboarding-block__question-skip" isLink>
+								<Link to="/design" className="onboarding-block__question-skip" isLink>
```

Comment:
Aside, I think we should rename the `className` since it's no longer just about skipping.